debian/install

```diff
@@ -0,0 +1 @@
+web usr/share/jellyfin/
```

Comment:
did you test that it would work?.. I mean, it would probably require setting up JF server files to point to this directory, right?

Reply:
Yes, I did test it out and it works fine as long as the /etc/default/jellyfin config is updated too. I believe there's a migration in the main PR, but if not I'll find and fix that.

Reply:
Okay, let's fix it in server build if it breaks. This path looks correct.